plugins/headlamp-plugin/template/package.json

```diff
@@ -29,6 +29,9 @@
     ]
   },
   "devDependencies": {
-    "@kinvolk/headlamp-plugin": "^$${headlamp-plugin-version}"
+    "@kinvolk/headlamp-plugin": "^$${headlamp-plugin-version}",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "@storybook/addon-storyshots": "6.4.13"
```

Comment:
We've been trying to avoid shipping dependencies beyond headlamp-plugin, so we'd not have to take care of keeping them under control. But if there's no workaround for it, I guess we have to use them..